flavors/publisher.go

```diff
@@ -88,7 +95,8 @@ func (p *Publisher) publish(events *[]beat.Event) {
 		return
 	}
 
-	p.log.Infof("Publishing %d events to elasticsearch", len(*events))
+	msg := fmt.Sprintf("Publishing %d events to elasticsearch", len(*events))
+	p.log.Infow(msg, ecsEventActionField, ecsEventActionValue, ecsEventCountField, len(*events))
```

Comment:
Kinda prefer this syntax, not sure what's considered best:
```suggestion
	p.log.With(ecsEventActionField, ecsEventActionValue, ecsEventCountField, len(*events)).Infof("Publishing %d events to elasticsearch", len(*events))
```

The benefit of this is we avoid a `fmt.Sprintf` call which will run even when the configured log level is higher than the one of this specific line.